src/io/flutter/run/daemon/DaemonApi.java

```diff
@@ -236,6 +203,64 @@ private Command takePending(int id) {
     }
   }
 
+  /**
+   * Parse the given string; if it's valid JSON, and a valid Daemon response of event, then return
```

Comment:
Not important probably but I find this description hard to follow... 

> "valid Daemon response of event" 

=> missing a word maybe?

Reply:
Or an extra one?

Reply:
updated the wording